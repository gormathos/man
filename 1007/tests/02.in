-29;99;70;-26;-39;47;21;-73;95;-43
10;48;62;24;-87;83;-84;3;-97;-85
73;-64;-92;77;-75;81;88;-31;-31;65
22;93.85;-2;-62;68;-24;-55;-47; ;99
35;-55;0;78;23;9M;44;49;10;73
25;-51;97;-54;100;-27;79;92;84;-27
6;46;42;52;-15;86;31;27;21;29
-3;84; ;72;-13;WQ;2;-58;76;48
-6;34;-22;-48;60;47;13;-3;-46;45
-9;-57;-63;-79;-5;74;-31;8;-85;5
