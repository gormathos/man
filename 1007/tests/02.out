-29;99;70;-26;-39;47;21;-73;95;-43
10;48;62;24;-87;83;-84;3;-97;-85
73;-64;-92;77;-75;81;88;-31;-31;65
25;-51;97;-54;100;-27;79;92;84;-27
6;46;42;52;-15;86;31;27;21;29
-6;34;-22;-48;60;47;13;-3;-46;45
-9;-57;-63;-79;-5;74;-31;8;-85;5
